src/Whoops/Resources/views/env_details.html.php

```diff
@@ -17,7 +17,7 @@
             <?php foreach ($data as $k => $value): ?>
               <tr>
                 <td><?php echo $tpl->escape($k) ?></td>
-                <td><?php echo $tpl->escape($tpl->dump($value)) ?></td>
+                <td><?php echo $tpl->dump($value) ?></td>
```

Comment:
as the string is now properly returned, it reveals a bug here... the escaping is not required/correct, as `dump()` returns html.